provisioning/resources/configs/compositions/docker-compose-aws.yml

```diff
@@ -46,7 +50,7 @@ services:
         awslogs-stream: "kibana"
 
   elasticsearch-loader-good:
-    image: snowplow/elasticsearch-loader:1.0.7
+    image: snowplow/elasticsearch-loader:2.0.8
```

Comment:
Did you try `2.0.8-distroless`?

Reply:
Do we have distroless for elasticsearch loader ? I couldn't see in [here](https://hub.docker.com/r/snowplow/elasticsearch-loader/tags)

Reply:
Sorry my mistake - you are right.  Elasticsearch loader is the only app for which we don't have a distroless image (yet).